js/src/ui/Form/Input/input.js

```diff
@@ -66,6 +45,7 @@ export default class Input extends Component {
       'default',
       'initial'
     ]),
+    fluid: PropTypes.bool,
```

Comment:
I'm going to add both than. Because it's easier to just lookup in semantics docs on what to do, but meanwhile obviously its good to be backward compatible.

I just don't see the value in using a built UI and not being able to consistently refer to the docs rather than have to go digging into ui->forms to remember.